providers/base/units/graphics/jobs.pxu

```diff
@@ -289,6 +289,72 @@ command:
  fi
 _summary: Check the OpenGL renderer (AMD GPU and DRI_PRIME=1, nvidia GPU and __NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia)
 
+unit: template
+template-resource: graphics_card
+plugin: shell
+category_id: com.canonical.plainbox::graphics
+id: graphics/{index}_glxgears_auto_{product_slug}
+flags: also-after-suspend
+requires: executable.name == 'glxgears'
+command:
+ # shellcheck disable=SC1091
+ prime_offload_tester.py -c glxgears -p {pci_device_name} -d {driver} -t 30
+_summary: Test that glxgears works for {vendor} {product}
```

Comment:
Since this job is gonna be running for 30 seconds, I would add an `estimated_duration: 30s` to these jobs.